### ISSUE TITLE
Support public stats or sharing of stats

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5574]
**Describe the problem you would like to solve**
Journals want some way to share their usage stats and editorial stats. There are several different kinds of sharing or public stats that may be requested:

- An always-available public URL with rolling up-to-date stats on some thing (eg - last 6 months X).
- A one-off public URL with the stats on some thing _as they are at the time the report is generated_ (eg - our average response time as generated on 21 Jan 2018).
- Same as the last, but in some kind of printable (eg - PDF or HTML print view) or easily distributable form (eg - charts rendered as images) for grant applications or other third-party uses.
- The ability to share (but not make public) one-off reports with a single user or user group to whom it is not normally available.
- The ability to share one-off reports with someone who is not registered with the journal (eg - Google Docs' shareable links for otherwise private documents).

**Describe the solution you'd like**
Each of these probably requires a separate implementation.

**Who is asking for this feature?**
Split from a discussion in [this issue](https://github.com/pkp/pkp-lib/issues/3365#issuecomment-365344412).

**Comment by bolelligallevi** [comment]
Hi @NateWr, we are using OJS 3.2 and OMP 3.1 and we would like to share usage stats of our journals and books... Something like this https://www.uclpress.co.uk/pages/statistics
There's some new feature in 3.3 or planned for future versions?
Best regards
Stefano

**Comment by NateWr** [comment]
No, this is not available or planned at this time. You can see what is planned for future versions in our [roadmap](https://pkp.sfu.ca/roadmap).

**Comment by bolelligallevi** [comment]
Thanks!
